### DISCUSSION TITLE
Making Tutorials

### DISCUSSION THREAD

**Discussion by Mouflon23** [FrancescoCeruti/linux-show-player#318] [General]
Hi, French guy here
Starting to use Linux Show Player. Really impressed of what is already done and I didn't see any tutorials on Youtube for this software ! Just videos of demonstration.

Should I make some or the project is already taken by someone else ?

**Comment by FrancescoCeruti** [comment]
Hi! 
Nothing "official" going on, so, If you want to, it would be very welcome 🚀
